CGMBLEKitUI/TransmitterSettingsNavigationController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TransmitterSettingsNavigationController.swift
+//  CGMBLEKitUI
+//
+//  Created by Nathaniel Hamming on 2021-01-15.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import HealthKit
+import LoopKit
+import LoopKitUI
+
+class TransmitterSettingsNavigationController: SettingsNavigationViewController, PreferredGlucoseUnitObserver {
```

Comment:
Recommend ditching this and using generic `CGMManagerSettingsNavigationController` described in my `LoopKit` PR comments.